src/java/org/apache/cassandra/service/CassandraDaemon.java

```diff
@@ -498,13 +499,13 @@ public void runStartupChecks()
     {
         try
         {
-            startupChecks.verify(DatabaseDescriptor.getStartupChecksOptions());
+            StartupChecksOptions startupChecksOptions = DatabaseDescriptor.getStartupChecksOptions();
+            startupChecks.verify(startupChecksOptions);
         }
         catch (StartupException e)
         {
             exitOrFail(e.returnCode, e.getMessage(), e.getCause());
         }
-
```

Comment:
I think we are brave enough to allow ourselves this blank like to be deleted, I am changing just this method and that line is redundant.